lib/elixir/pages/Writing Documentation.md

```diff
@@ -48,6 +48,38 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
 
     def size(map)
 
+## Functions with multiple clauses
+
+When documenting a function with multiple clauses, it is recommended to add the documentation
+and typespecs to the function head rather than adding @doc and @spec to each clause of
+the function.
```

Comment:
Could you not wrap lines and place this sentence over a single line? Just to be coherent with the style of the rest of this document :smiley: